model/src/w3gdatmd.F90

```diff
@@ -46,28 +46,28 @@ MODULE W3GDATMD
 !/    30-Oct-2009 : Implement curvilinear grid type.    ( version 3.14 )
 !/                  (W. E. Rogers & T. J. Campbell, NRL)
 !/    29-Oct-2010 : Implement unstructured grids        ( version 3.14.1 )
-!/                  (A. Roland and F. Ardhuin) 
+!/                  (A. Roland and F. Ardhuin)
```

Comment:
Most changes are white space or style and to me seem unrelated to the PR.   I'm wondering if it's worth keeping these changes in the PR or if it would be better to revert this file to its original state which would help keep this branch and develop in sync.